src/main/java/org/testng/TestRunner.java

```diff
@@ -254,6 +254,9 @@ private void init() {
 
     initListeners();
     addConfigurationListener(m_confListener);
+    for (IConfigurationListener cl : m_configuration.getConfigurationListeners()) {
```

Comment:
@missedone - Since I am not that familiar with the eclipse plugin, I thought I would ask you instead..

* Does this change handle all the listeners that are wired in via the service loaders ? 
* I was under the impression that execution would always start from the constructor of `TestNG` and in that process listeners would all get wired in (be it via the suite xml or be it via service loaders). If that's correct, then why do we need to call this config listener addition explicitly here.